forage-android/src/main/java/com/joinforage/forage/android/core/ui/element/ForagePanElement.kt

```diff
@@ -258,20 +270,21 @@ abstract class ForagePanElement @JvmOverloads constructor(
     // Therefore we expose novel set listener methods instead of
     // overriding the convention setOn*Listener
     override fun setOnFocusEventListener(l: SimpleElementListener) {
-        manager.setOnFocusEventListener(l)
+        onFocusEventListener = l
         restartFocusChangeListener()
     }
     override fun setOnBlurEventListener(l: SimpleElementListener) {
-        manager.setOnBlurEventListener(l)
+        onBlurEventListener = l
         restartFocusChangeListener()
     }
-    override fun setOnChangeEventListener(l: StatefulElementListener<PanElementState>) {
-        manager.setOnChangeEventListener(l)
+    override fun setOnChangeEventListener(l: StatefulElementListener<PanEditTextState>) {
+        onChangeEventListener = l
     }
 
-    override fun getElementState(): PanElementState {
-        return manager.getState()
-    }
+    override fun getElementState(): PanEditTextState = PanEditTextState.from(
+        focusState,
+        inputState
+    )
```

Comment:
These methods expose the `PanElementState` to the public surface area, which forces us to make `PanElementState` `public` 😕